test/unit/test_mail_quoting.js

```diff
@@ -393,8 +415,9 @@ commonCase('Quoting', function(T) {
         eCheck.expect(tdef.chunks[i], tdef.chunks[i+1]);
       }
       var roundtrip = true;
-      if (tdef.hasOwnProperty('roundtrip'))
+      if ('roundtrip' in tdef) {
```

Comment:
Just curious about this change: my naive reading of the file seemed like hasOwnProperty would be a better fit, no prototypes involved.